ee/maintained-apps/inputs/homebrew/scripts/google_chrome_install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+installer -pkg "$INSTALLER_PATH" -target /
```

Comment:
The pkg handles closing google chrome if it's already open and all the other fancy stuff our typical install script does?